qdrant-landing/content/documentation/cloud/create-cluster.md

```diff
@@ -91,6 +91,26 @@ Once provisioned, you can access your cluster on ports 443 and 6333 (REST) and 6
 
 You should now see the new cluster in the **Clusters** menu.
 
+## Creating a Production-Ready Cluster
+
+To create a production-ready cluster, you need to ensure the following:
+
+**High Availability**
+
+Your cluster should have at least 3 nodes, and each collection should have a replication factor of at least 2. This ensures that if one node fails, or is being restarted due to maintenance, version upgrades or scaling operations, the cluster remains fully operational. You can ensure this by checking the **High Availability** checkbox when creating a cluster.
```

Comment:
'This ensures that is one node fails, or is restarted due to maintenance, a version upgrade, or a scaling operation, that the cluster remains fully operational.'